src/components/custom/signup.js

```diff
@@ -89,6 +89,14 @@ const Signup = (props) => {
         const cookies = getCookiesFields()
         const cookies_objects = getCookiesObject(cookies)
         const cookies_value = getDataObjFromCookies(cookies_objects, cookies)
+        const token = queryParams.get('t')
+
+        if (token == null) {
+            Cookies.remove('affiliate_tracking')
+            cookies_value.utm_campaign = 'null' //passing null as the cookies takes the affiliates token value when signed up without affiliate token
+            cookies_value.utm_medium = 'null'
+            cookies_value.utm_source = 'null'
```

Comment:
We don't want the  `utm_campaign` and `utm_medium` show up in the API request. We can assign `null` instead of 'null' to eliminate them

Reply:
validation fails if we pass null.so that's the reason i passed as string null